cli/format/format_extended.go

```diff
@@ -132,14 +132,7 @@ func extendedFormatAnnotations(labels models.LabelSet) string {
 func extendedFormatMatchers(matchers models.Matchers) string {
 	output := []string{}
 	for _, matcher := range matchers {
-		output = append(output, extendedFormatMatcher(*matcher))
+		output = append(output, simpleFormatMatcher(*matcher))
 	}
 	return strings.Join(output, " ")
```

Comment:
I think we should also utilize how `labels.Matchers.String` formats multiple matchers, i.e. `{foo="bar", dings="bums"}` rather than `foo="bar" dings="bums"` as it is done now. And I think now that you have implemented `labelsMatcher`, it's very easy:

```Go
lms := labels.Matchers{}
for _, matcher := range matchers {
	lms = append(lms, labelsMatcher(*matcher)
}
return lms.String()
```